bundler/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghcr.io/dependabot/dependabot-updater-core
+
+RUN apt-get update \
+  && apt-get upgrade -y \
+  && apt-get install -y --no-install-recommends \
+    curl
+
+USER dependabot
+
+COPY --chown=dependabot:dependabot bundler/helpers /opt/bundler/helpers
+RUN bash /opt/bundler/helpers/v1/build \
+  && bash /opt/bundler/helpers/v2/build
+
+COPY --chown=dependabot:dependabot bundler /home/dependabot/bundler
```

Comment:
I think we need to install bundler v1 somewhere since we didn't transfer it into the updater-core image, since it's only needed by bundler ecosystem.

```
gem install bundler -v $BUNDLER_V1_VERSION --no-document
```

Reply:
Hmmm the bundler v1 tests all passed 🤔 .

Reply:
I think global Bundler 1 installation is no longer needed. The native helper builder takes care of it.

Reply:
interesting that the ci passes without it..

Reply:
I created #6493 to verify my claim :)